components/ProposedGoalPreview/index.js

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+const StyledTitle = styled.p`
+  margin-bottom: 0;
+`;
+
+export default function ProposedGoalPreview({ image, title }) {
+  return (
+    <>
+      <div>
+        <Image src={image} alt={title} width={40} height={40} />
+        <StyledTitle>{title}</StyledTitle>
+      </div>
+    </>
```

Comment:
There's no need for a React Fragment (<>...</>) because you are returning a single parent JSX element that already wraps all other elements.

```suggestion
      <div>
        <Image src={image} alt={title} width={40} height={40} />
        <StyledTitle>{title}</StyledTitle>
      </div>
```